obs2ioda-v2/src/cxx/netcdf_attribute.cc

```diff
@@ -45,6 +45,15 @@ namespace Obs2Ioda {
         }
     }
 
+    int netcdfPutAttInt1D(
+        int netcdfID, const char *attName, const int *attValue,
+        const char *varName, const char *groupName, const int len
```

Comment:
Just a stylistic consideration: would adding the `len` argument immediate before or after the `attValue` argument make it clearer that these to arguments are connected? That would also be more in line with the netCDF [`nc_put_att_int`](https://docs.unidata.ucar.edu/netcdf-c/4.9.2/group__attributes.html#gab1089956b01ce5b40da6ace82a2d3b30) interface.

Also, renaming the `len` argument to `attLen` makes it clearer that this is the length of the attribute in my opinion.